yarn-project/ethereum/src/config.ts

```diff
@@ -236,15 +210,15 @@ export const getRewardBoostConfig = () => {
 const LocalEntryQueueConfig = {
   bootstrapValidatorSetSize: 0n,
   bootstrapFlushSize: 0n,
-  normalFlushSizeMin: 48n, // will effectively be bounded by maxQueueFlushSize
+  normalFlushSizeMin: 48n,
   normalFlushSizeQuotient: 2n,
   maxQueueFlushSize: 32n,
 };
 
 const StagingPublicEntryQueueConfig = {
   bootstrapValidatorSetSize: 48n,
-  bootstrapFlushSize: 32n, // will effectively be bounded by maxQueueFlushSize
-  normalFlushSizeMin: 32n,
+  bootstrapFlushSize: 48n,
+  normalFlushSizeMin: 1n,
```

Comment:
Do we want to set the flush size to 48 as in StagingIgnition in https://github.com/AztecProtocol/aztec-packages/pull/17364? Not saying we should, just asking.